device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py

```diff
@@ -60,7 +60,7 @@
                  'x86_64-mlnx_msn2410-r0': 2, 'x86_64-mlnx_msn2010-r0': 3, 'x86_64-mlnx_msn3420-r0': 5,
                  'x86_64-mlnx_msn3700-r0': 0, 'x86_64-mlnx_msn3700c-r0': 0, 'x86_64-mlnx_msn3800-r0': 4,
                  'x86_64-mlnx_msn4410-r0': 0, 'x86_64-mlnx_msn4600-r0': 4, 'x86_64-mlnx_msn4600c-r0': 4, 
-                 'x86_64-mlnx_msn4700-r0': 0}
+                 'x86_64-mlnx_msn4700-r0': 0, 'x86_64-nvidia_sn5600-r0': 4}
```

Comment:
@raphaelt-nvidia  Are you sure about pointing to "4" ?

Reply:
Discussed this with Lior.  4 is the index into this table:

# port_position_tuple = (PORT_START, QSFP_PORT_START, PORT_END, PORT_IN_BLOCK, EEPROM_OFFSET)\
port_position_tuple_list = [(0, 0, 31, 32, 1), (0, 0, 15, 16, 1), (0, 48, 55, 56, 1),
                            (0, 18, 21, 22, 1), (0, 0, 63, 64, 1), (0, 48, 59, 60, 1)]

#4 describes 64 ports all of which are QSFP.

root@r-moose-simx-161:/home/admin# show interfaces status 
  Interface                            Lanes    Speed    MTU    FEC    Alias    Vlan    Oper    Admin    Type    Asym PFC
-----------  -------------------------------  -------  -----  -----  -------  ------  ------  -------  ------  ----------
  Ethernet0                  0,1,2,3,4,5,6,7     400G   9100    N/A     etp1  routed      up       up     N/A         N/A
  Ethernet8            8,9,10,11,12,13,14,15     400G   9100    N/A     etp2  routed      up       up     N/A         N/A
Ethernet16          16,17,18,19,20,21,22,23     400G   9100    N/A     etp3  routed      up       up     N/A         N/A
Ethernet24          24,25,26,27,28,29,30,31     400G   9100    N/A     etp4  routed      up       up     N/A         N/A
Ethernet32          32,33,34,35,36,37,38,39     400G   9100    N/A     etp5  routed      up       up     N/A         N/A
Ethernet40          40,41,42,43,44,45,46,47     400G   9100    N/A     etp6  routed      up       up     N/A         N/A
Ethernet48          48,49,50,51,52,53,54,55     400G   9100    N/A     etp7  routed      up       up     N/A         N/A
Ethernet56          56,57,58,59,60,61,62,63     400G   9100    N/A     etp8  routed      up       up     N/A         N/A
Ethernet64          64,65,66,67,68,69,70,71     400G   9100    N/A     etp9  routed      up       up     N/A         N/A
Ethernet72          72,73,74,75,76,77,78,79     400G   9100    N/A    etp10  routed      up       up     N/A         N/A
Ethernet80          80,81,82,83,84,85,86,87     400G   9100    N/A    etp11  routed      up       up     N/A         N/A
Ethernet88          88,89,90,91,92,93,94,95     400G   9100    N/A    etp12  routed      up       up     N/A         N/A
Ethernet96      96,97,98,99,100,101,102,103     400G   9100    N/A    etp13  routed      up       up     N/A         N/A
Ethernet104  104,105,106,107,108,109,110,111     400G   9100    N/A    etp14  routed      up       up     N/A         N/A
Ethernet112  112,113,114,115,116,117,118,119     400G   9100    N/A    etp15  routed      up       up     N/A         N/A
Ethernet120  120,121,122,123,124,125,126,127     400G   9100    N/A    etp16  routed      up       up     N/A         N/A
Ethernet128  128,129,130,131,132,133,134,135     400G   9100    N/A    etp17  routed      up       up     N/A         N/A
Ethernet136  136,137,138,139,140,141,142,143     400G   9100    N/A    etp18  routed      up       up     N/A         N/A
Ethernet144  144,145,146,147,148,149,150,151     400G   9100    N/A    etp19  routed      up       up     N/A         N/A
Ethernet152  152,153,154,155,156,157,158,159     400G   9100    N/A    etp20  routed      up       up     N/A         N/A
Ethernet160  160,161,162,163,164,165,166,167     400G   9100    N/A    etp21  routed      up       up     N/A         N/A
Ethernet168  168,169,170,171,172,173,174,175     400G   9100    N/A    etp22  routed      up       up     N/A         N/A
Ethernet176  176,177,178,179,180,181,182,183     400G   9100    N/A    etp23  routed      up       up     N/A         N/A
Ethernet184  184,185,186,187,188,189,190,191     400G   9100    N/A    etp24  routed      up       up     N/A         N/A
Ethernet192  192,193,194,195,196,197,198,199     400G   9100    N/A    etp25  routed      up       up     N/A         N/A
Ethernet200  200,201,202,203,204,205,206,207     400G   9100    N/A    etp26  routed      up       up     N/A         N/A
Ethernet208  208,209,210,211,212,213,214,215     400G   9100    N/A    etp27  routed      up       up     N/A         N/A
Ethernet216  216,217,218,219,220,221,222,223     400G   9100    N/A    etp28  routed      up       up     N/A         N/A
Ethernet224  224,225,226,227,228,229,230,231     400G   9100    N/A    etp29  routed      up       up     N/A         N/A
Ethernet232  232,233,234,235,236,237,238,239     400G   9100    N/A    etp30  routed      up       up     N/A         N/A
Ethernet240  240,241,242,243,244,245,246,247     400G   9100    N/A    etp31  routed      up       up     N/A         N/A
Ethernet248  248,249,250,251,252,253,254,255     400G   9100    N/A    etp32  routed      up       up     N/A         N/A
Ethernet256  256,257,258,259,260,261,262,263     400G   9100    N/A    etp33  routed      up       up     N/A         N/A
Ethernet264  264,265,266,267,268,269,270,271     400G   9100    N/A    etp34  routed      up       up     N/A         N/A
Ethernet272  272,273,274,275,276,277,278,279     400G   9100    N/A    etp35  routed      up       up     N/A         N/A
Ethernet280  280,281,282,283,284,285,286,287     400G   9100    N/A    etp36  routed      up       up     N/A         N/A
Ethernet288  288,289,290,291,292,293,294,295     400G   9100    N/A    etp37  routed      up       up     N/A         N/A
Ethernet296  296,297,298,299,300,301,302,303     400G   9100    N/A    etp38  routed      up       up     N/A         N/A
Ethernet304  304,305,306,307,308,309,310,311     400G   9100    N/A    etp39  routed      up       up     N/A         N/A
Ethernet312  312,313,314,315,316,317,318,319     400G   9100    N/A    etp40  routed      up       up     N/A         N/A
Ethernet320  320,321,322,323,324,325,326,327     400G   9100    N/A    etp41  routed      up       up     N/A         N/A
Ethernet328  328,329,330,331,332,333,334,335     400G   9100    N/A    etp42  routed      up       up     N/A         N/A
Ethernet336  336,337,338,339,340,341,342,343     400G   9100    N/A    etp43  routed      up       up     N/A         N/A
Ethernet344  344,345,346,347,348,349,350,351     400G   9100    N/A    etp44  routed      up       up     N/A         N/A
Ethernet352  352,353,354,355,356,357,358,359     400G   9100    N/A    etp45  routed      up       up     N/A         N/A
Ethernet360  360,361,362,363,364,365,366,367     400G   9100    N/A    etp46  routed      up       up     N/A         N/A
Ethernet368  368,369,370,371,372,373,374,375     400G   9100    N/A    etp47  routed      up       up     N/A         N/A
Ethernet376  376,377,378,379,380,381,382,383     400G   9100    N/A    etp48  routed      up       up     N/A         N/A
Ethernet384  384,385,386,387,388,389,390,391     400G   9100    N/A    etp49  routed      up       up     N/A         N/A
Ethernet392  392,393,394,395,396,397,398,399     400G   9100    N/A    etp50  routed      up       up     N/A         N/A
Ethernet400  400,401,402,403,404,405,406,407     400G   9100    N/A    etp51  routed      up       up     N/A         N/A
Ethernet408  408,409,410,411,412,413,414,415     400G   9100    N/A    etp52  routed      up       up     N/A         N/A
Ethernet416  416,417,418,419,420,421,422,423     400G   9100    N/A    etp53  routed      up       up     N/A         N/A
Ethernet424  424,425,426,427,428,429,430,431     400G   9100    N/A    etp54  routed      up       up     N/A         N/A
Ethernet432  432,433,434,435,436,437,438,439     400G   9100    N/A    etp55  routed      up       up     N/A         N/A
Ethernet440  440,441,442,443,444,445,446,447     400G   9100    N/A    etp56  routed      up       up     N/A         N/A
Ethernet448  448,449,450,451,452,453,454,455     400G   9100    N/A    etp57  routed      up       up     N/A         N/A
Ethernet456  456,457,458,459,460,461,462,463     400G   9100    N/A    etp58  routed      up       up     N/A         N/A
Ethernet464  464,465,466,467,468,469,470,471     400G   9100    N/A    etp59  routed      up       up     N/A         N/A
Ethernet472  472,473,474,475,476,477,478,479     400G   9100    N/A    etp60  routed      up       up     N/A         N/A
Ethernet480  480,481,482,483,484,485,486,487     400G   9100    N/A    etp61  routed      up       up     N/A         N/A
Ethernet488  488,489,490,491,492,493,494,495     400G   9100    N/A    etp62  routed      up       up     N/A         N/A
Ethernet496  496,497,498,499,500,501,502,503     400G   9100    N/A    etp63  routed      up       up     N/A         N/A
Ethernet504  504,505,506,507,508,509,510,511     400G   9100    N/A    etp64  routed      up       up     N/A         N/A

Reply:
OK. Thanks